pkg/gitconfig/configs.go

```diff
@@ -145,6 +145,22 @@ func (cs *Configs) loadGlobalConfigs() string {
 		locs = append(locs, filepath.Join(appdir.UserHome(), cs.GlobalConfig))
 	}
 
+	// if we already have a global config we can just reload it instead of trying all locations
+	if !cs.global.IsEmpty() {
+		if p := cs.global.path; p != "" {
```

Comment:
super-nit: I think you could combine both if statements to remove a level of nesting: `if p := cs.global.path; !cs.global.IsEmpty() && p != ""` (not sure if that's much better).

Reply:
Oh. Yes, then you are right of course.